CMakeLists.txt

```diff
@@ -118,6 +123,9 @@ add_subdirectory(protobuf)
 add_subdirectory(schemachange)
 add_subdirectory(sockpool)
 add_subdirectory(sqlite)
+if(WITH_TCL)
+  add_subdirectory(tcl)
+endif()
```

Comment:
This does seem to work.  It causes the directory to be skipped unless the feature is enabled.